impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/finalBeanClass/AfterBeanDiscoveryObserver.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.decorators.custom.broken.finalBeanClass;
+package org.jboss.cdi.tck.tests.full.decorators.broken.finalBeanClass;
```

Comment:
`decorators.custom` should IMHO be moved to `full.decorators.custom`, not `full.decorators`.

Reply:
Done